tests/ccl_test_cmblens.c

```diff
@@ -0,0 +1,91 @@
+#include "ccl.h"
+#include "../include/ccl_params.h"
+#include "ctest.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <math.h>
+#include <string.h>
+
+#define CLS_TOLERANCE 1E-3
+#define CLS_FRACTION 1E-3
+
+CTEST_DATA(cls) {
+  double Omega_c;
+  double Omega_b;
+  double h;
+  double A_s;
+  double n_s;
+  double sigma_8;
+};
+
+CTEST_SETUP(cls) {
+  data->Omega_c = 0.30;
+  data->Omega_b = 0.00;
+  data->h = 0.7;
+  data->A_s = 2.1e-9;
+  data->sigma_8=0.8;
+  data->n_s = 0.96;
+}
+
+static int linecount(FILE *f)
+{
+  //////
+  // Counts #lines from file
+  int i0=0;
+  char ch[1000];
+  while((fgets(ch,sizeof(ch),f))!=NULL) {
+    i0++;
+  }
+  return i0;
+}
+
+static void compare_cls(struct cls_data * data)
+{
+  int status=0;
+
+  ccl_configuration config = default_config;
+  config.transfer_function_method = ccl_bbks;
+  config.matter_power_spectrum_method = ccl_linear;
+  ccl_parameters params = ccl_parameters_create_flat_lcdm(data->Omega_c,data->Omega_b,data->h,
+							  data->A_s,data->n_s, &status);
+  params.Omega_k=0;
+  params.Omega_g=0;
+  params.Omega_n_rel=0;
+  params.Omega_l = 1.0 - params.Omega_m;
+  params.sigma_8=data->sigma_8;
```

Comment:
This is just a comment in general for several of the test functions: why do we need data->sigma_8 having provided data->A_s? Is params.sigma_8 being populated by the correct value corresponding to the A_s from the input?

Reply:
Only for CLASS, but not for E&H or BBKS